play/src/front/Connection/RoomConnection.ts

```diff
@@ -658,6 +716,19 @@ export class RoomConnection implements RoomConnection {
                     this._xmppSettingsMessageStream.next(message.xmppSettingsMessage);
                     break;
                 }
+                case "mutedMessage": {
+                    notificationPlayingStore.playNotification(
+                        get(LL).notification.askToMuteMicrophone(),
+                        "audio-mute.svg"
+                    );
+                    requestedMicrophoneState.disableMicrophone();
+                    break;
+                }
+                case "mutedVideoMessage": {
+                    notificationPlayingStore.playNotification(get(LL).notification.askToMuteCamera(), "camera-off.png");
+                    requestedCameraState.disableWebcam();
+                    break;
+                }
```

Comment:
Same comment as above. Apply inversion of control, move specific code out of this class.